sdf/1.8/1_7.convert

```diff
@@ -1,3 +1,9 @@
 <convert name="sdf">
```

Comment:
Blocking item: Should remove debug-based shims. (Just putting it here for review-based tracking)

Reply:
Removed 3e0cd5d70fce3d7dbe32746c2f3ce526ab059baa